ParamConverter/DataTransferObjectParamConverter.php

```diff
@@ -496,4 +499,25 @@ protected function validate($object, Request $request, array $options): void
             $violations
         );
     }
+
+    /**
+     * @param Request $request
+     */
+    private function extractDataFromMultipartBody(Request $request): void
+    {
+        if (explode(';', $request->headers->get('CONTENT_TYPE', ''))[0] !== 'multipart/form-data') {
```

Comment:
Too many instruction at once, set some variables before to make it easier to understand